samples/Paging/Program.cs

```diff
@@ -7,12 +7,13 @@
 using Microsoft.Azure.CosmosRepository.Paging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Paging.Specifications;
 
 ConfigurationBuilder configuration = new();
 
 ServiceProvider provider = new ServiceCollection().AddCosmosRepository(options =>
     {
-        options.CosmosConnectionString = Environment.GetEnvironmentVariable("CosmosConnectionString");
+        options.CosmosConnectionString = "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
```

Comment:
Wow, probably not an ideal place to have a hard coded string that contains sub key. If you haven't already, you should remove this immediately - and then go and delete your corresponding resource (or at the very least regenerate the keys).

Reply:
Oh will cleanup. but it's no big deal from security aspect since this is the default and only possible accountkey for the cosmos db emulator.
https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21#authenticate-requests

Reply:
For the future you can set env variables for a test runner, I use rider and you can do it there but fairly sure you can do the same for VS 2022 with a run settings file or something like that, @IEvangelist may know more 😃